contracts/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "kava-contracts",
+  "version": "0.0.1",
+  "author": "Kava Labs",
+  "private": true,
+  "description": "Solidity contracts for Kava Blockchain",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "build": "npm run clean && npm run compile && npm run ethermint-json",
```

Comment:
as mentioned in the readme, this is the command that will compile the contract and convert it to the necessary json format for use in evmutil.